packages/algorand-js/src/internal/interpreter/interpreter.ts

```diff
@@ -0,0 +1,33 @@
+import type { execParams } from "algob/src/types";
+import { assert } from "chai";
+
+import { IStack, Stack } from "../../lib/stack";
+import type { AppArgs, Operator, StackElem } from "../types";
+
+export class Interpreter {
+  readonly _stack: IStack<StackElem>;
+
+  constructor () {
+    this._stack = new Stack<StackElem>();
+  }
+
+  /**
+ * Description: this function executes set of Operator[] passed after
+ * parsing teal code
+ * @param {execParams} txn : Transaction parameters for current txn
+ * @param {Logic[]} logic : set of Logic objects
+ * @param {AppArgs} args : argument array passed to the current transaction
+ * @returns {boolean} : transaction accepted/rejected based on asc logic
```

Comment:
I think this returns linter error - we don't need to do type annotation in comments, because we already have them in the function definition. If there is no linter error then leave it.

Reply:
Does not throw a linter error. I will remember this when writing doc string for functions in the future. Leaving it as the same for now.